File: spotsense/src/main/java/com/spotsense/data/network/loggingInterceptor/I.java
package com.spotsense.data.network.loggingInterceptor;

import android.util.Log;

class I {
    static void Log(int type, String tag, String msg) {
        switch (type) {
            case Log.VERBOSE:
//                Log.v(tag, msg);
                break;
            case Log.DEBUG:
//                Log.d(tag, msg);
                break;
            case Log.ERROR:
                Log.e(tag, msg);
                break;
            case Log.INFO:
//                Log.i(tag, msg);
                break;
            case Log.WARN:
//                Log.w(tag, msg);
                break;
            case Log.ASSERT:
//                Log.wtf(tag, msg);
                break;
        }
    }
}

File: spotsense/src/main/java/com/app/spotsense/network/loggingInterceptor/I.kt
package com.app.spotsense.network.loggingInterceptor

object I {
    @JvmStatic
    fun Log(type: Int, tag: String?, msg: String?) {
        when (type) {
            android.util.Log.VERBOSE -> {}
            android.util.Log.DEBUG -> {}
            android.util.Log.ERROR -> android.util.Log.e(tag, msg!!)
            android.util.Log.INFO -> {}
            android.util.Log.WARN -> {}
            android.util.Log.ASSERT -> {}
        }
    }
}